<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>881</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lbl_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>861</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>409</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="edt_data">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>9</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>409</y>
      <width>691</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="edt_file">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>611</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_frame">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>41</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>41</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_play">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>44</y>
       <width>62</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/play_icon.png</normaloff>img/play_icon.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+P</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>67</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame stop</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_frame_stop">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>67</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_record">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>44</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/record_icon.png</normaloff>img/record_icon.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+R</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_step">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_record_step">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/record_icon.png</normaloff>img/record_icon.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>553</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Step</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="action_open"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
